pkgs/by-name/q2/q2pro/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  pkg-config,
+  ninja,
+  zlib,
+  libpng,
+  libjpeg,
+  curl,
+  SDL2,
+  openalSoft,
+  libogg,
+  libvorbis,
+  libXi,
+  wayland,
+  wayland-protocols,
+  libdecor,
+  ffmpeg,
+  wayland-scanner,
+  makeWrapper,
+  versionCheckHook,
+  x11Support ? stdenv.isLinux,
+  waylandSupport ? stdenv.isLinux,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "q2pro";
+  version = "r3510";
```

Comment:
version should not prefix with a letter, see https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning